contracts/bonding/BondingManager.sol

```diff
@@ -814,21 +785,23 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         return activeTranscoderSet[_round].isActive[_transcoder];
     }
 
-    /*
+    /**
      * @dev Return whether a transcoder is registered
      * @param _transcoder Transcoder address
      */
     function isRegisteredTranscoder(address _transcoder) public view returns (bool) {
         return transcoderStatus(_transcoder) == TranscoderStatus.Registered;
     }
 
-    /*
+    /**
      * @dev Remove transcoder
      */
     function resignTranscoder(address _transcoder) internal {
         uint256 currentRound = roundsManager().currentRound();
         if (activeTranscoderSet[currentRound].isActive[_transcoder]) {
-            // If transcoder is active for the round set it as inactive
+            // Decrease total active stake for the round
```

Comment:
Whenever we resign a transcoder we can use this function which will not only remove the transcoder from the pool, but also check if the transcoder is active for the round and if so, set decrease the total active stake for the current round and set the transcoder as inactive